helm/cosmo/charts/controlplane/templates/delete-user.yaml

```diff
@@ -0,0 +1,116 @@
+{{ if .Values.jobs.deleteUser.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ include "controlplane.fullname" . }}-delete-user-{{ now | unixEpoch }}"
+  labels:
+    {{- include "controlplane.labels" . | nindent 4 }}
+  annotations:
+    # Run the job before installing the helm chart
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    # Support for k14s.io. This annotation ensure that the job is run after the postgresql and keycloak deployment
+    kapp.k14s.io/change-rule.postgres: "upsert after upserting cosmo.apps.postgresql.wundergraph.com/deployment"
+    {{- if .Values.global.keycloak.enabled }}
+    kapp.k14s.io/change-rule.keycloak: "upsert after upserting cosmo.apps.keycloak.wundergraph.com/deployment"
+    {{- end }}
+spec:
+  backoffLimit: 10
+  parallelism: 1
+  ttlSecondsAfterFinished: 360
```

Comment:
We should increase the ttl for all jobs. 5min is very low. 2d is fine. This allows to inspect all jobs for debugging purpose.